mkdocs.yml

```diff
@@ -42,7 +42,7 @@ nav:
   #    - NCP Perspective: cave/ncp-perspective.md
       - NSHARP: cave/nsharp.md
       - WarnGen Walkthrough: cave/warngen.md
-      # - GOES 16/17: cave/goes-16-17-satellite.md
+      - GOES 16/17: cave/goes-16-17-satellite.md
```

Comment:
change to `- GOES East/West: cave/goes-16-17-satellite.md`

The part on the left of the : is the title that shows up in the table of contents on our website, the part to the right of the : is the filename to reference (which we didn't change, but maybe could/should?)